zetaclient/zetacore/background_threads.go

```diff
@@ -0,0 +1,162 @@
+package zetacore
```

Comment:
`backgroud_threads` would be too abstract for a name
We should have filename reflecting content
- tss_update_listeners
- etc..